src/styles/header.scss

```diff
@@ -0,0 +1,16 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+  &__logo {
+    position: relative;
```

Comment:
it's a pretty strange way to align elements in the header. It also applies to the right margin in the nav.

The simple way is to wrap the header content in a container and align it the way you want it.